vars/launchBuildFromRepo.groovy

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/groovy
+
+def call(ctx, jobName, jobId, buildName, fromTag, toTag, sourceImage, sourcePath) {
```

Comment:
as mentioned on bjns, jobName, jobId, buildName, sourceImage can be obtained from the ctx
inputs should be fromTag, toTag, dockerfilePath